.circleci/run-firesim-tests.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# turn echo on and error on earliest command
+set -ex
+
+# get remote exec variables
+SCRIPT_DIR="$( cd "$( dirname "$0" )" && pwd )"
+source $SCRIPT_DIR/defaults.sh
+
+export PATH=$LOCAL_VERILATOR_DIR/install/bin:$PATH
+export FIRESIM_ENV_SOURCED=1
+
+SIMULATION_ARGS="${mapping[$1]}"
+
+run_test_suite () {
```

Comment:
Why not the ScalaTests here?

Reply:
This way of running it is easier to fit into the existing CI method, which is to separate building the verilator model from running the tests. But I suppose I should make the set of tests run here match what's run in ScalaTests.

Reply:
The scala tests check additional things (e.g., that the trace widget is properly capturing the instruction trace. 

Maybe the best approach is to modify the scala tests to use an existing simulator if it's already compiled? That way we can still have the CI generation and simulator stages?

Reply:
We resolved to modify the scala tests in the future to permit reusing a precompiled simulator.

Reply:
You don't have to follow the model... The original purpose of splitting up the build from the tests was so that BOOM could split its assembly tests and bmark tests and run them in parallel in two containers. Since it looks like Chipyard is using CI as a smoke test then we can just have this script run the build and tests.